src/resources/Change/GeneralPartnershipResource.ts

```diff
@@ -8,10 +8,11 @@ export const GeneralPartnershipResource: ResourceIF = {
   displayName: GetCorpFullDescription(CorpTypeCd.PARTNERSHIP),
   nameRequestType: NameRequestEntityTypes.GP,
   addressLabel: 'Business Addresses',
-  filingData: {
+  filingData: [{
+    filingTypeCode: FilingCodes.CONVERSION,
     entityType: CorpTypeCd.PARTNERSHIP,
-    filingTypeCode: FilingCodes.CHANGE_OF_REGISTRATION
-  },
+    priority: false
```

Comment:
priority is a required prop in FilingDataIF and would get compile warning if I did not add them. So defaulted them to false.